app/stac_api/views.py

```diff
@@ -152,13 +153,24 @@ def get_queryset(self):
         date_time = self.request.query_params.get('datetime', None)
 
         if bbox:
-            raise NotImplementedError('bbox query parameter not yet implemented')
+            queryset = self.filter_by_bbox(queryset, bbox)
 
         if date_time:
             queryset = self.filter_by_datetime(queryset, date_time)
 
         return queryset
 
+    def filter_by_bbox(self, queryset, bbox):
+        try:
+            logger.debug(bbox)
```

Comment:
I would be a bit more verbose here in the log or remove it, e.g. 'filter with bbox %s', bbox.

Reply:
this was a develop logger - I removed it